src/librustdoc/html/highlight/fixtures/union.html

```diff
@@ -1,8 +1,8 @@
-<span class="kw">union</span> <span class="ident">Foo</span> {
-    <span class="ident">i</span>: <span class="ident">i8</span>,
-    <span class="ident">u</span>: <span class="ident">i8</span>,
+<span class="kw">union </span><span class="ident">Foo {
+    i: i8,
+    u: i8,
 }
```

Comment:
It's the part up to debate since most of the gain comes from being very lax on `ident`. I kinda like having the "ident" class since it still provides an information, but maybe we could completely get rid of it. Not sure.

An opinion maybe @jsha?

Reply:
Technically we have a style that matches .ident but it's pretty narrowly scoped:

```
pre.rust .attribute, pre.rust .attribute .ident { color: #C82829; }
```

I agree with @notriddle: we're not using the `ident` class for anything right now, so let's get rid of it.